### ISSUE TITLE
Custom/ConcurrentMonotonicTime has the wrong namespace - should be Performance

### ISSUE THREAD

**Issue by pboling** [standardrb/standard-custom#3]
```
standard-custom-1.0.0/config/ruby-2.2.yml: Custom/ConcurrentMonotonicTime has the wrong namespace - should be Performance
```

**Comment by pboling** [comment]
NOTE: this was fixed in https://github.com/standardrb/standard-custom/commit/d1463642babf24871e9c011d8caa3678e1cd0233 but there hasn't been a new release yet

**Comment by searls** [comment]
Released in 1.0.1
